### PR TITLE
Adding example for Dynamic Ceph

### PR DESCRIPTION
**Pull request by ahardin-rh**
Continuation of https://github.com/openshift/openshift-docs/pull/4446

FYI @donnydavis 

Preview build: http://file.rdu.redhat.com/~ahardin/07262017/donnydavis-dynamic-ceph/install_config/storage_examples/ceph_rbd_dynamic_example.html

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@openshift/team-documentation Please review :heart_decoration:

File: install_config/storage_examples/ceph_rbd_dynamic_example.adoc [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+
+This topic provides an end-to-end example of using an existing Ceph cluster as
+an {product-title} dynamic persistent store. It is assumed that a working Ceph
+cluster is already set up. If not, consult the
+link:https://access.redhat.com/products/red-hat-ceph-storage[Overview of Red Hat
+Ceph Storage].
+
+xref:../persistent_storage/persistent_storage_ceph_rbd.adoc#install-config-persistent-storage-persistent-storage-ceph-rbd[Persistent
+Storage Using Ceph Rados Block Device] provides an explanation of persistent
+volumes (PVs), persistent volume claims (PVCs), and how to use Ceph Ceph Rados
```
Comment (mburke5678) [inline install_config/storage_examples/ceph_rbd_dynamic_example.adoc]: Ceph Ceph?

Comment (ahardin-rh) [inline install_config/storage_examples/ceph_rbd_dynamic_example.adoc]: :scream_cat: Good catch. Thanks!

File: install_config/storage_examples/ceph_rbd_dynamic_example.adoc [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+  adminSecretNamespace: kube-system  <4>
+  pool: kube  <5>
+  userId: kube  <6>
+  userSecretName: ceph-secret-user  <7>
+----
+<1> Ceph monitors, comma delimited. It is required.
+<2> Ceph client ID that is capable of creating images in the pool. Default is `admin`.
+<3> Secret Name for `adminId`. It is required. The provided secret must have type `kubernetes.io/rbd`.
+<4> The namespace for `adminSecret`. Default is `default`.
+<5> Ceph RBD pool. Default is `rbd`.
```
Comment (mburke5678) [inline install_config/storage_examples/ceph_rbd_dynamic_example.adoc]: Is this the default RBD pool you recommended against using earlier?

File: install_config/storage_examples/ceph_rbd_dynamic_example.adoc [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+# oc create -f ceph-pod1.yaml
+pod "ceph-pod1" created
+
+#verify pod was created
+# oc get pod
+NAME        READY     STATUS    RESTARTS   AGE
+ceph-pod1   1/1       Running   0          2m
+                      <1>
+----
+<1> After a minute or so, the pod will be in the `Running` state.
```
Comment (mburke5678) [inline install_config/storage_examples/ceph_rbd_dynamic_example.adoc]: s/After a minute or so/After approximately a minute (??)

**Comment by mburke5678** [comment]
Couple o' comments @ahardin-rh

File: install_config/storage_examples/ceph_rbd_dynamic_example.adoc (hunk 2 above)
Comment (ahardin-rh) [inline install_config/storage_examples/ceph_rbd_dynamic_example.adoc]: @mburke5678 Good catch. I'll emphasize that again here. 
@donnydavis Are there any other recommended values besides `kube`?

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../ install_config/storage_examples/ceph_rbd_dynamic_example.adoc#install-config-storage-examples-ceph-rbd-dynamic-example[Persistent Storage Examples -> Complete Example Using Ceph RBD for Dynamic Provisioning]
|New topic outlining a complete example using Ceph RBD for dynamic provisioning.
%

**Comment by donnydavis** [comment]
Looks reallye great. Sorry I lost track of this. Pesky day jobs

**Pull request merged**